### ISSUE TITLE
EJP XML manifest items to ignore

### ISSUE THREAD

**Issue by Melissa37** [elifesciences/package-ejp-raw-output-zip#2]
## Problem / Motivation
- WHO: Production 
- WHEN: Between Export from EJP/xPub and delivery to production vendor
- WHERE: In eLife bot processes
- WHAT: Ignore unnecessary data when re-building the XML manifest file OR remove from the manifest file before sending on to production
- WHY: It has the potential to cause confusion with the new vendor (they could accidentally add it to the production XML without us seeing it in a UI anywhere and it being delivered downstream and be in our final XML). This is part of the process to retain control in eLife in the step between one system and the next process. EJP could not exclude this content from the export but we don't use it/want it.

## Proposed solution
Reference other tickets in this repo for the whole process, this is a specific task in the conversion process.
Following lines of XML should be ignored and not pulled further into the next steps of the process:

```<journal-id journal-id-type="hwp">eLife</journal-id>```

```<open-access>YES</open-access>```

``` <alt-title></alt-title>```

```<issue>0</issue>```

```<pub-date pub-type="epub"><year>2018</year></pub-date>```

```<date date-type="rev-recd"><day>25</day><month>10</month><year>2018</year></date>```

```<custom-meta><meta-name>fast-track</meta-name><meta-value>no</meta-value></custom-meta>```

```<custom-meta><meta-name>export-timing</meta-name><meta-value>ready-for-production</meta-value></custom-meta>```

``` <custom-meta><meta-name>eLife Digest</meta-name><meta-value>Yes</meta-value></custom-meta>```
        
```<custom-meta><meta-name>Insight question</meta-name><meta-value>Yes</meta-value></custom-meta>```

```<volume>```

## Assumptions

- Assumption - eLife will never need these fields in their final XML or require them for processing content through production. Exception - volume - but the vendor will need to add the value so better they start from scratch so we know they've added it
- Assumption - PoA publication date will not be known at the time of export and so building systems to add this date in at this stage is not the most efficient way to deal with this.
- Assumption - all content is fast track so we do not use this field now and probably not in the future

## Clarification needed 
- Question - Ask Features the function of the eLife digest and Insight question and whether they will be necessary for the production process.
---

## Tasks
- [ ] @gnott  to provide

## Technical notes

- Don't write the code so it expects these fields as they will not exist in the next submission system output

## User interface / Wireframes
NA

@gnott  @eLifeProduction

**Comment by gnott** [comment]
A sample I have also has this tag, would you want it in or out?

```
<volume>0</volume>
```

**Comment by gnott** [comment]
Can these particular tags be removed from the manifest XML file as part of the packaging process so in the final output they do no appear at all? Or do you intend for them to remain but just do not consider their values in the packaging process?

**Comment by Melissa37** [comment]
>A sample I have also has this tag, would you want it in or out? ```<volume>```

Good point, out - because the production process is the only point when the volume will be known - volumes change between 31st Dec and 1st Jan.
I will update the ticket.

>Can these particular tags be removed from the manifest XML file as part of the packaging process so in the final output they do no appear at all? Or do you intend for them to remain but just do not consider their values in the packaging process?

I suspect you will end up writing a new manifest file, based on the full set of requirements as they come. But if not, they should be completely removed and not sent on to the production process.
